alacritty_terminal/src/input.rs

```diff
@@ -392,6 +398,7 @@ impl Action {
                 ctx.spawn_new_instance();
             },
             Action::None => (),
+            Action::Unset => unreachable!(),
```

Comment:
I'm not sure if I `unreachable` is the correct choice here, but I think it does make some sense and should help us detect errors early. Though if we don't detect them, it's possible that the effects are more severe.

In any case, if we do choose to use `unreachable!` here, I'd recommend passing at least a short message to it to make it absolutely clear that it's intentional. That will also make debugging trivial even without any backtraces.

Reply:
`Unset` cannot be added by users, because they are removed (by definition). If that code is reached it can "only happen" if it comes from a hard-coded binding that's absolutely useless.

Reply:
I'm aware of that. It still adds potential for a crash when that is absolutely not necessary. If we ever hit this, we could handle this like the `None` action without any issues.

Reply:
So then an error message for `uncreachable!` is enough? Or choose "`()`"?

Reply:
I'd choose an error message for `unreachable!` and rely on people reporting the crash if it should ever happen. It's an easy fix if it does, so it's not a bad idea to know about it.